index/index_utils.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2023 Princess B33f Heavy Industries / Dave Shanley
+// SPDX-License-Identifier: MIT
+
+package index
+
+import (
+    "gopkg.in/yaml.v3"
+    "net/http"
+    "strings"
+    "time"
+)
+
+func isHttpMethod(val string) bool {
+    switch strings.ToLower(val) {
```

Comment:
nit: kind of unreadable; I guess these match up to [these](https://github.com/pb33f/libopenapi/blob/7c6507488d9734b961d2d49514fefdb5350c8359/index/index_model.go#L258)?

Reply:
Old refactored code won't fix.